frontend/src/generated/graphql.tsx

```diff
@@ -4754,6 +4795,64 @@ export type GetDeviceTypeListQueryResult = Apollo.QueryResult<
   GetDeviceTypeListQuery,
   GetDeviceTypeListQueryVariables
 >;
+export const GetSupportedDiseasesDocument = gql`
+  query getSupportedDiseases {
+    supportedDiseases {
+      internalId
+      name
+    }
+  }
+`;
```

Comment:
non-blocking question: where are updates to this file getting generated from?

Reply:
This file gets generated by `yarn codegen`, these particular changes come from the changes in the file `frontend/src/app/supportAdmin/DeviceType/operations.graphql` - our code generator looks for all files named `operations.graphql` and generates hooks for us to use in our components!